### PR TITLE
Backend health command changed on new Varnish versions

### PR DESCRIPTION
**Pull request by shaun-rutherford**
## Pull Request Checklist

**Is this in reference to an existing issue?** No

#### General

- [ x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ x] RuboCop passes

- [ ] Existing tests pass

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

#### Purpose
New Versions of Varnish changed the debug.health command to backend.list, added code to check for either.

### CODE REVIEW

File: CHANGELOG.md
Comment (majormoses) [inline CHANGELOG.md:8]: generally it is best to leave the version bump to maintainers as there is no guarantee on response times or merge order.

**Review by majormoses — APPROVED** [review]
Thanks for the contribution!
LGTM

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-varnish/versions/1.1.2

File: CHANGELOG.md
Comment (shaun-rutherford) [reply CHANGELOG.md:8]: Ah gotcha, sorry first time making a contribute to a public project.  I was trying to follow the guides which states to update the change log, I'll make sure to wait for response in the future :)

Comment (majormoses) [reply CHANGELOG.md:8]: Totally understandable and welcome to open source!

Ya we need to really write up better guidelines, we want a changelog entry (and we are slightly opinionated flavor of keepachangelog) but leaving the actual version bumping to the maintainers for the above reason.

**Pull request merged**
